pkg/kubectl/cmd/version.go

```diff
@@ -138,7 +138,11 @@ func (o *VersionOptions) Complete(cmd *cobra.Command) error {
 	return nil
 }
 
-func (o *VersionOptions) Validate() error {
+func (o *VersionOptions) Validate(args []string) error {
+	if len(args) > 0 {
+		return fmt.Errorf("not require arguments")
```

Comment:
change to `not expect too many arguments` ?

Reply:
Done,  thanks @dixudx PTAL